addons/binding/org.openhab.binding.leapmotion/src/main/java/org/openhab/binding/leapmotion/LeapMotionBindingConstants.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014,2018 by the respective copyright holders.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
```

Comment:
Actually I was trying to build the project to see the SAT results. That would have probably shown that we still use EPL-1.0 instead of EPL-2.0 in this repo. ;-) Maybe you can check the SAT results for more issues?